### PR TITLE
show end pose published to CANDIDATE_ROBOT_ENDPOSE

### PR DESCRIPTION
**Pull request by MarcoCar**

### CODE REVIEW

File: src/python/ddapp/teleoppanel.py [hunk 1]
```diff
@@ -765,6 +768,15 @@ def searchFinalPoseClicked(self):
...
             self.panel.showPose(self.constraintSet.endPose)
         app.displaySnoptInfo(info)
+    
+    def onCandidateEndPose(self, msg):
+        if not self.getCheckboxState(self.ui.finalPosePlanningOptions) and not self.ui.eeTeleopButton.checked:
+            translation = (msg.pose.translation.x, msg.pose.translation.y, msg.pose.translation.z)
+            rotation = transformUtils.quaternionToRollPitchYaw((msg.pose.rotation.w, msg.pose.rotation.x, msg.pose.rotation.y, msg.pose.rotation.z))
+            pose = list(translation)
+            pose.extend(rotation)
+            pose.extend(list(msg.joint_position))
```
Comment (patmarion) [inline src/python/ddapp/teleoppanel.py]: this is not the correct way to construct a pose.  You should use:


pose = robotstate.convertStateMessageToDrakePose(msg)

That function will do the correct mapping from joint names to positions using the msg.joint_name array.

**Comment by MarcoCar** [comment]
Thanks @patmarion. It should be ok now.

File: src/python/ddapp/teleoppanel.py [hunk 2]
```diff
@@ -12,7 +12,8 @@
 from ddapp import drcargs
 from ddapp import affordanceurdf
 from ddapp.roboturdf import HandFactory
-import ddapp.applogic as app
+from ddapp import lcmUtils
+import drc
```
Comment (patmarion) [inline src/python/ddapp/teleoppanel.py]: There's probably a merge conflict at these import statements.  You'll have to rebase onto master or merge master.  For the import, change "ddapp" to "director".

Also, if you could, I usually do "import drc as lcmdrc" and then later refer to lcmtypes as lcmdrc.robot_state_t

**Comment by patmarion** [comment]
Great, I will merge it.  Unfortunately, the recent merge made a conflict with this one, sorry!

**Comment by MarcoCar** [comment]
@patmarion I hope this is mergeable now.

**Pull request merged**